### DISCUSSION TITLE
How to get context authorization headers from nextauth with Apollo clients??

### DISCUSSION THREAD

**Discussion by rahXephonz** [nextauthjs/next-auth#4074] [Help] [answered]
Hello,

I have questions about nextauth and want implement it with Apollo client for example

I use nextjs for client
And for the backend use Prisma, Postgresql, Apollo server and GraphQL. So I want ask

How to get a authorization headers that come from nextjs and Apollo client with same JWT and verify it
I think nextauth headers come from cookie not local storage. So how to implement that? I was search any references and didn't found with clear solutions.

**Comment by rahXephonz** [comment, marked as answer]
@balazsorban44 hello sir, are you have an idea or references? 🥺🥺

**Reply by balazsorban44** [reply]
Yep, docs: https://next-auth.js.org/getting-started/example#using-nextauthjs-callbacks

**Reply by rahXephonz** [reply]
Thanks a lot for references
